File: src/main/java/um/haberes/report/model/haberes/core/CargoDto.java
package um.haberes.report.model.haberes.core;

import com.fasterxml.jackson.annotation.JsonFormat;
import java.math.BigDecimal;
import java.time.OffsetDateTime;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CargoDto {

    private Long cargoId;

    private Long legajoId;

    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ssXX", timezone = "UTC")
    private OffsetDateTime fechaAlta;

    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ssXX", timezone = "UTC")
    private OffsetDateTime fechaBaja;

    private Integer dependenciaId;

    private Integer categoriaId;

    private int jornada = 0;

    private int presentismo = 0;

    private BigDecimal horasJornada = BigDecimal.ZERO;

}

File: src/main/java/um/haberes/report/kotlin/dto/haberes/core/CargoDto.kt
package um.haberes.report.kotlin.dto.haberes.core

import com.fasterxml.jackson.annotation.JsonFormat
import java.math.BigDecimal
import java.time.OffsetDateTime

data class CargoDto (
    var cargoId: Long? = null,

    var legajoId: Long? = null,

    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ssZ", timezone = "UTC")
    var fechaAlta: OffsetDateTime? = null,

    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ssZ", timezone = "UTC")
    var fechaBaja: OffsetDateTime? = null,

    var dependenciaId: Int? = null,
    var categoriaId: Int? = null,
    var jornada: Int = 0,
    var presentismo: Int = 0,
    var horasJornada: BigDecimal = BigDecimal.ZERO,

    ){
}